src/mrsimulator/methods/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+import warnings
+
+warnings.warn(
+    (
+        "Importing library methods from `mrsimulator.methods` is deprectated and will "
+        "be removed in the next version. Please import library methods from the "
+        "`mrsimulator.method.lib` module."
+    ),
+    Warning,
+)
+
+from mrsimulator.method.lib import BlochDecayCentralTransitionSpectrum  # noqa:F401
+from mrsimulator.method.lib import BlochDecayCTSpectrum  # noqa:F401
+from mrsimulator.method.lib import BlochDecaySpectrum  # noqa:F401
+from mrsimulator.method.lib import FiveQ_VAS  # noqa:F401
+from mrsimulator.method.lib import SevenQ_VAS  # noqa:F401
+from mrsimulator.method.lib import ThreeQ_VAS  # noqa:F401
+from mrsimulator.method.lib import SSB2D  # noqa:F401
+from mrsimulator.method.lib import ST1_VAS  # noqa:F401
+from mrsimulator.method.lib import ST2_VAS  # noqa:F401
```

Comment:
Move the warnings after the import statements.

Instead of importing all classes, you may use 
```py
from mrsimulator.method.lib import *
```

The only public objects in the `lib.__init__.py` file are the class imports, so it should be fine.

Reply:
There is no need for methods to be a folder with the `__init__.py` file. You can create a file called `methods.py`.